### ISSUE TITLE
Eslint 6.8.0 does not follow symlinked directory in Atom but it does in VSCode

### ISSUE THREAD

**Issue by yahyaensi** [eslint/eslint#13174]
Eslint version: 6.8.0
Atom: 1.45.0
linter: 2.3.1
linter-eslint 8.5.5

I have a symlinked directory, eslint doesn't report errors in its files at all.

Could you add a property that force eslint to report issues in a symlinked directory?

Thanks

**Comment by mdjermanovic** [comment]
Hi @yahyaensi, thanks for the issue!

Do you see the expected errors when you run ESLint from the command line?

**Comment by yahyaensi** [comment]
hi @mdjermanovic , thanks for your replay.
The problem is related to the Atom paramter  "Exclude VCS Ignored Paths" that should be deactivated in order to allow the Atom plugin "linter" to process folders and files declared in eslintignore (in fact, the symlinked directory is declared in eslintignore)

**Comment by mdjermanovic** [comment]
So, the problem is when you deactivate "Exclude VCS Ignored Paths" you still don't see errors in editor?

I think this is more a question for [AtomLinter/linter-eslint](https://github.com/AtomLinter/linter-eslint) repository, but if it helps you can also try with this setting:

Packages/linter-eslint/Uncommon/Don't use .eslintignore files

config:

```
"linter-eslint":
    advanced:
      disableEslintIgnore: true
```
